incubator/drone/Chart.yaml

```diff
@@ -0,0 +1,15 @@
+name: drone
+version: 0.1.0
+description: Drone CI
+keywords:
+- continuous-deployment
+- continuous-integration
+- docker
+- drone
+- drone.io
+- go
+sources:
+- http://readme.drone.io/
+maintainers:
+- name: Matthew Fisher
+  email: mfisher@deis.com
```

Comment:
- Can you change your name to your github handle? Recent thing we started doing.
- Change your email address